ui/components/Metadata.tsx

```diff
@@ -41,40 +40,62 @@ function Metadata({ metadata = [], labels = [], className }: Props) {
         </Link>
       );
   });
+  return metadataCopy;
+};
 
+const MetadataSection: React.FC<{ title: string; items: [string, any][] }> = ({
```

Comment:
Is `any` type really valid here? If I pass in `{ foo: { bar: 'baz' }}`, will the component render OK?

Reply:
It was `any` bc we're turning some of the values into links - I can change it into `string | JSX`

Reply:
For the sake of accuracy, I like `string | JSX`. That way the next person has an idea of what they can pass in.